pkg/metrics/metrics.jsx

```diff
@@ -194,7 +194,7 @@ const HISTORY_METRICS = [
 ];
 
 function debug() {
-    if (window.debugging == "all" || window.debugging == "metrics")
+    if (window.debugging == "all" || window.debugging?.includes("metrics"))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18492-20230313-062223-0ed5c399-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)